README.md

```diff
@@ -175,7 +175,12 @@ banana.load(messages, 'es' );
 
 #### Load the messages for many locales at once
 
-If you think it is convinient to load all messages for all locales at once, you can do this as follows. Here the messages are keyed by locale.
+> While it is possible to store all translations in a single file, we recommend using separate files for each language.
+> This approach makes it easier to find and update >translations. It also simplifies tracking changes via version control systems (VCS) and
+> enables multiple translators or teams to work on different language files simultaneously without conflicts. Additionally, using separate files
+> allows loading only the translations for the currently selected language, improving performance by avoiding the need to load all translations.
```

Comment:
Some notes:
* Not sure it makes easier to find and update translations (there is stray > on the line.
* I think we can expect people to know version control systems so I see no use introducing the acronym for it
* You might want to specifically call out that this is requirement for translatewiki.net

Reply:
Done.